forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java

```diff
@@ -39,7 +39,7 @@ private static Hash commitSquashed(Repository localRepo, Hash finalHead, Author
                                 committer.name(), committer.email(), ZonedDateTime.now(), List.of(targetHash(localRepo)), localRepo.tree(finalHead));
     }
 
-    private final static Pattern mergeSourcePattern = Pattern.compile("^Merge ([-/\\w:+]+)$");
+    private final static Pattern mergeSourcePattern = Pattern.compile("^Merge ([-/.\\w:+]+)$");
```

Comment:
This will match any character, which may not be what you intend. Shouldn't that be `\\.` instead of just `.`?

Reply:
My understanding is that the only characters you need to escape inside a character class are ^, -, ] or \, everything else is a literal. Also Intellij gave me a warning when I tried to escape it saying it was unnecessary.

Reply:
I stand corrected. I now see that `.` is a literal dot when it is inside `[]` (when not inside `[]` it matches any char).